.github/workflows/benchmark.yml

```diff
@@ -147,9 +135,11 @@ jobs:
           MSSQL_PASSWORD: MsSql!1234
           TEST_CLICKHOUSE_IMAGE: "clickhouse/clickhouse-server:23-alpine"
           TEST_COUCHBASE_IMAGE: "couchbase:enterprise-7.6.5"
+          TEST_DYNAMODB_IMAGE: amazon/dynamodb-local:2.6.0
```

Comment:
Leave this as latest. Maintaining track of patch releases is a pain 😂

Reply:
The issue with latest is that it could be anything causing tests to break. OTOH I see the value in not maintaining the versions of all supported technologies. Changing it in a follow-up commit.